moto/apigateway/models.py

```diff
@@ -72,7 +72,7 @@ def cloudformation_type():
 
     @classmethod
     def create_from_cloudformation_json(
-        cls, resource_name, cloudformation_json, region_name
+        cls, resource_name, cloudformation_json, region_name, **kwargs
```

Comment:
Custom resources also need to know the LogicalResourceId/StackId/ResourceType. Passing in kwargs was the least invasive signature change that leaves the door open for further extensions